automated-comments/javascript/gigasecond/prefer_number_comprehension.md

```diff
@@ -0,0 +1,2 @@
+Rewrite the literal `%{literal}` using `Math.pow` or `10 ** n` in order to make
```

Comment:
Is this a disapproval, or an approval? If it is the latter, then maybe rephrase a bit to make it a suggestion instead of an order :)

Reply:
I think it currently is a `disapproval` 😓 😅

Reply:
Alright! Then ignore my comment.